snapcraft/internal/lifecycle.py

```diff
@@ -326,8 +326,12 @@ def _tar_filter(tarinfo):
 
 def containerbuild(step, project_options, output=None, args=[]):
     config = snapcraft.internal.load_config(project_options)
+    remote = os.environ.get('SNAPCRAFT_CONTAINER_BUILDS')
+    if remote in ['1', '0', None, '']:
+        remote = None
```

Comment:
I find it weird that when the env var is set to 1, then it's the same as not setting it.
Could you raise an error if an invalid value is set?